docs/networks/staking/06-technical-overview.md

```diff
@@ -82,9 +82,9 @@ Node operators need to determine the role of node they will be running
 (Collection, Consensus, Execution, Verification, or Access).
 
 <Callout type="warning">
-  NOTE: Access Nodes are eligible to stake but will not receive rewards for
-  their stake. Please register as a different node type if you would like to
-  receive rewards.
+  NOTE: Access Nodes are eligible to stake and have a staking minimum of 50 FLOW,
```

Comment:
```suggestion
  NOTE: Access Nodes are eligible to stake and have a staking minimum of 100 FLOW,
```
It says 100 [further up](https://github.com/onflow/docs/pull/834/files#diff-76b3f193c7b94d760c760c05093eddf4256c2cb7134913664afc69ea14cae71dR128)